components/zcash_protocol/src/txid.rs

```diff
@@ -65,10 +65,4 @@ impl TxId {
         writer.write_all(&self.0)?;
         Ok(())
     }
-
-    /// Returns true when the txid consists of all zeros; this only occurs for coinbase
-    /// transactions.
-    pub fn is_null(&self) -> bool {
```

Comment:
Do not remove this from the public API. `TxId` nullability is an annoying thing inherited from Bitcoin, but one we need to preserve due to consensus (i.e. that an all-zeros txid cannot exist outside of coinbase). The fact that this is not used internally to the librustzcash workspace does not mean it is unused. This is also an unnecessary breaking change to `zcash_protocol` that contributes to this PR not being additive.